code/client/src/core/modules/human.cpp

```diff
@@ -122,6 +126,31 @@ namespace MafiaMP::Core::Modules {
                         tracking.human->SetRot({newRot.x, newRot.y, newRot.z, newRot.w});
                     }
                 }
+
+                const auto humanPtr = tracking.human;
+                const auto hp       = updateData->_healthPercent;
+                const auto nickname = humanData->nickname;
+                gApplication->GetImGUI()->PushWidget([humanPtr, hp, nickname]() {
+                    const auto displaySize               = ImGui::GetIO().DisplaySize;
+                    auto gameCamera                      = SDK::ue::game::camera::C_GameCamera::GetInstanceInternal();
+                    auto camera                          = gameCamera->GetCamera(SDK::ue::game::camera::E_GameCameraID::CAMERA_PLAYER_MAIN_1);
+                    auto camPos                          = camera->GetPos();
+                    static const auto headBoneHash       = SDK::ue::sys::utils::C_HashName::ComputeHash("Head");
```

Comment:
Why static?

Reply:
there's no difference, still has to be static.

Reply:
putting it outside lambda means extra code to capture static var, which is not required.